### ISSUE TITLE
Text can have both header & list mark at the same time

### ISSUE THREAD

**Issue by trompx** [Canner/canner-slate-editor#25]
Hey @chilijung,

Again I don't know if this behaviour was intentional but it differs from the one at http://slatejs.org/#/rich-text, where when clicking the list mark when a header is focused, it looses the header mark and transforms into a list, while your version allow both header & list mark.

**Comment by chilijung** [comment]
Yeah, I expect this behavior. Any reason that header should not be a list? I'll leave this issue here, to hear how others think!
